steam/client/__init__.py

```diff
@@ -297,9 +297,8 @@ def send(self, message, body_params=None):
         """
         if not self.connected:
             self._LOG.debug("Trying to send message when not connected. (discarded)")
-        else:
-            if body_params and isinstance(message, MsgProto):
-                proto_fill_from_dict(message.body, body_params)
+        elif body_params and isinstance(message, MsgProto):
+            proto_fill_from_dict(message.body, body_params)
 
             CMClient.send(self, message)
```

Comment:
this actually changes the logic as `CMClient.send(self, message)` would only be called when the if condition matches, while the original version ran it in both cases in the else block

Reply:
gotcha. From my brief look it looks like you're right. Next time I'm at my computer i'll take a closer like and most likely revert that part.

Reply:
thanks for calling me out on that one mate. I've reverted that change now.